### PR TITLE
negative and positive cMVAv2 conf files for BTV

### PR DESCRIPTION
**Pull request by jpata**
Does not affect reco sequences.
Nota: work in progress, do not test yet.
Includes merges

```
cms-btv-pog:FixTrackQuality-from-CMSSW_7_6_1
cms-btv-pog:FixSignedMuonTaggers_from-CMSSW_7_6_1
```

### CODE REVIEW

File: RecoBTag/Combined/src/CombinedMVAV2JetTagComputer.cc [hunk 1]
```diff
@@ -99,6 +99,13 @@ float CombinedMVAV2JetTagComputer::discriminator(const JetTagComputer::TagInfoHe
   inputs["Jet_CSVIVF"]   = (*(computers[2]))( TagInfoHelper(ivfTagInfos) );
   inputs["Jet_SoftMu"]   = (*(computers[3]))( TagInfoHelper(smTagInfos) );
   inputs["Jet_SoftEl"]   = (*(computers[4]))( TagInfoHelper(seTagInfos) );
+  //std::cout << "JP=" << inputs["Jet_JP"]
```
Comment (slava77) [inline RecoBTag/Combined/src/CombinedMVAV2JetTagComputer.cc]: please drop the commented out code, or if you still need it, replace with LogDebug

File: SimTracker/TrackHistory/python/TrackQuality_cff.py [hunk 2]
```diff
@@ -11,6 +11,8 @@
 		associatePixel = cms.bool(True),
 		associateStrip = cms.bool(True),
 		associateRecoTracks = cms.bool(True),
-		ROUList = copy.deepcopy(tabh.trackAssociatorByHits.ROUList)
+		ROUList = copy.deepcopy(tabh.trackAssociatorByHits.ROUList),
+                pixelSimLinkSrc = cms.InputTag("simSiPixelDigis"),
```
Comment (slava77) [inline SimTracker/TrackHistory/python/TrackQuality_cff.py]: why is this file changed?
Please remove changes here or explain why needed

**Comment by slava77** [comment]
@jpata 
Please make a PR in 80X.
We will integrate it first and then go back to 76X

**Comment by jpata** [comment]
OK! I made this PR to have a reference point for the comments while I started developing on 76X.

File: SimTracker/TrackHistory/python/TrackQuality_cff.py (hunk 2 above)
Comment (jpata) [inline SimTracker/TrackHistory/python/TrackQuality_cff.py]: this is from merging cms-btv-pog:FixTrackQuality-from-CMSSW_7_6_1. @imarches is this required for 80X? What is the usual procedure in case of dependent merges?

Comment (ferencek) [inline SimTracker/TrackHistory/python/TrackQuality_cff.py]: @jpata, there is no need to carry around these other commits as part of this PR even if your newly added code functionally depends on them (actually, the TrackHistory fixes are totally decoupled from this PR). These other commits are already or will be integrated through separate PRs. So as long as these dependent PRs can be merged separately, I would keep them separate.

**Comment by jpata** [comment]
https://github.com/cms-sw/cmssw/pull/13009
